dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -504,7 +549,7 @@ else if (techniqueName.equals("selection")) {
                     double[] fig18_values_array = currProj.get_fig18_values();
                     double[] fig18_percents_array = currProj.get_fig18_percents();
                     // original values, index should be i=0, i+=2
-                    if (resolveDependences == null) { // original
+                    if (resolveDependences == null) { // orig, unenhanced
```

Comment:
The variable name here really shouldn't be resolveDependences since that corresponds to a different argument for the scripts. Change this variable to isEnhancedTech or isUsingDTFile